mappings/net/minecraft/client/gl/DummyUniform.mapping

```diff
@@ -0,0 +1,10 @@
+CLASS none/bwj net/minecraft/client/gl/DummyUniform
+	METHOD a setFloat1 (F)V
```

Comment:
*Could* just be named `setFloat` and then use overloading instead. Similarly with other methods.